### ISSUE TITLE
Dismiss, Snooze actions by volume buttons

### ISSUE THREAD

**Issue by mizzunet** [yuriykulikov/AlarmClock#392]
# Dismiss, Snooze actions by volume buttons
## Summary

Press volume button to dismiss or snooze

## Motivation

Easy to do those action(?)

## Description

While alarm playing, press volume button to either dismiss or snooze which set from Setiings

## Drawbacks, risks and assumptions

I'm not sure about this

## Rationale and alternatives

Google's Clock app has similar feature.

Thank you ;)

**Comment by kolanse** [comment]
Hi @yuriykulikov  I am back now . work has lessened . can i look into this .. ?

**Comment by yuriykulikov** [comment]
Sure, it would be great 👍

**Comment by kolanse** [comment]
Should i pull off develop branch ? @yuriykulikov

**Comment by yuriykulikov** [comment]
Yes, develop branch is the branch from which we build apps for Google play and f-droid.

**Comment by theMr17** [comment]
Hello, is this token still not done, if so can I do it? @yuriykulikov

**Comment by yuriykulikov** [comment]
Of course, the contribution would be very welcome 🤗

**Comment by arseniiv** [comment]
There are drawbacks if this is not an opt-in behavior. Pressing volume buttons can be done even less consciously than pressing an area on the screen (if you can’t do it in one try, you surely can find it by continuously bashing fingers into the phone—but finding the volume button solely by touch is even easier).
